wpeverest/user-registration
# UR-1686 Enhance - Email marketing integration menu in form settings

### All Submissions:






### Changes proposed in this Pull Request:
This PR introduces a list of integration settings in Form Settings, Which incorporates all integration settings under one in pro which were displayed individually previously and also displays addon premium condition popup with video for free when clicked.

### How to test the changes in this Pull Request:
1. Activate a free version and verify list is showing and the proper message is shown while clicking add-ons on integration settings in form settings.
2. In the pro version check if integrations are properly working.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Email marketing integration menu in form settings.
